build.sbt

```diff
@@ -12,26 +12,26 @@ version := "1.0-SNAPSHOT"
  * SUB-PROJECT DEFINITIONS
  */
 
-lazy val commonWithTests: ClasspathDep[ProjectReference] = common % "compile;test->test;it->it"
+lazy val commonWithTests: ClasspathDep[ProjectReference] = common % "compile;test->test;it->it;pt->pt;at->at;at->it"
```

Comment:
Yes and no.

If `inConfig(PlayTest)(Defaults.testSettings)` is present the tests will not be executed twice, but by default there are executed in each scope.